WooCommerce/Classes/POS/Controllers/PointOfSaleItemsController.swift

```diff
@@ -30,23 +29,21 @@ class PointOfSaleItemsController: PointOfSaleItemsControllerProtocol {
     }
 
     @MainActor
-    func loadInitialItems(base: ItemListBaseItem) async {
+    func reloadItems(base: ItemListBaseItem) async {
         switch base {
         case .root:
-            await loadInitialRootItems()
+            await loadRootItems()
         case .parent(let parent):
-            await loadInitialChildItems(for: parent)
+            await loadChildItems(for: parent)
```

Comment:
Yeah, probably `loadItems` for the function given this naming. The only downside is that it could be misunderstood as though it'll load all items, but I think it's probably clear enough, no need for objective C levels of verbosity.